spa/src/pages/ConfigSteps/OrgsContainer/index.tsx

```diff
@@ -49,8 +56,54 @@ const OrganizationsList = ({
 	resetCallback: (args: boolean) => void;
 	connectingOrg: (org: GitHubInstallationType) => void;
 }) => {
-	const [clickedOrg, setClickedOrg] = useState<GitHubInstallationType | undefined>(undefined);
-	const canConnect = (org: GitHubInstallationType) => !org.requiresSsoLogin && !org.isIPBlocked && org.isAdmin;
+	const [isScrolledToBottom, setIsScrolledToBottom] = useState(false);
+	const [isListScrollable, setIsListScrollable] = useState(false);
+	const contentRef = useRef<HTMLDivElement | null>(null);
+	const hasScrolledYetRef = useRef(false);
+	const [clickedOrg, setClickedOrg] = useState<
+		GitHubInstallationType | undefined
+	>(undefined);
+
+	const canConnect = (org: GitHubInstallationType) =>
+		!org.requiresSsoLogin && !org.isIPBlocked && org.isAdmin;
+
+	const checkIsListScrollable = () => {
+		let isListScrollable = false;
```

Comment:
\nit
This feels like a util/helper method, which only checks if a DOM element is scrollable or not. I'd suggest to move out the `setIsListScrollable` inside the useEffect and make this method into a util method which return boolean.